chpl/chpl-service/src/main/java/gov/healthit/chpl/app/questionableActivity/QuestionableActivityReportApp.java

```diff
@@ -59,57 +63,65 @@
     private static final int ACTIVITY_CERT_STATUS_CHANGE_REASON_COL = 11;
     private static final int ACTIVITY_REASON_COL = 12;
     private Date startDate, endDate;
-    
+
     protected QuestionableActivityDAO qaDao;
     protected NotificationDAO notificationDao;
 
+    /**
+     * Constructor.
+     */
     public QuestionableActivityReportApp() {
     }
-    
-    public QuestionableActivityReportApp(Date startDate, Date endDate) {
+
+    /**
+     * Constructor with start/end dates.
+     * @param start start date of report
+     * @param end end date of report
+     */
+    public QuestionableActivityReportApp(final Date start, final Date end) {
         this();
-        this.startDate = startDate;
-        this.endDate = endDate;
+        this.startDate = start;
+        this.endDate = end;
     }
 
-    protected void initiateSpringBeans(AbstractApplicationContext context) throws IOException {
+    protected void initiateSpringBeans(final AbstractApplicationContext context) throws IOException {
```

Comment:
I'm sure this is fine I'm just curious why you added final to these variable defs

Reply:
Because checkstyle kept telling me I needed to